android/assets/jsons/translations/Traditional_Chinese.properties

```diff
@@ -1324,7 +1324,7 @@ Unhappiness modifier = 不滿修正
 Bonus vs. Barbarians = 對戰蠻族時攻擊加成
 Barbarian spawning delay = 蠻族生成延遲
  # Requires translation!
-Bonus starting units = 
+Bonus starting units = 蠻族起始單位
```

Comment:
![image](https://user-images.githubusercontent.com/100454479/173194213-b31fa5f4-e786-4749-909e-cff3802176c6.png)
Bonus doesn't have to be the babarions.
```suggestion
Bonus starting units = 額外起始單位
```

Reply:
Thanks, i see.